### PR TITLE
Adds flashcards to Glossary & makes small corrections

### PR DESCRIPTION
**Pull request by darigovresearch**
# What does this PR do?

Flashcards have been made using the glossary as a starting point and are now linked at the start of the glossary. Other small corrections & standardisations have also been made for consistency.

This pull requests follows from the discussion in issue #8932

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors which may be interested in your PR.
@sgugger

### CODE REVIEW

File: docs/source/glossary.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Glossary
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+These definitions have also been put into a form which can be easily learnt/revised using `Anki <https://apps.ankiweb.net/>`_ a cross platform app specifically designed for long term knowledge retention.
+
+- `Introductory video on how to use the flashcards <https://www.youtube.com/watch?v=Dji_h7PILrw>`_
+- `Hugging Face Transformers Glossary Flashcards <https://www.darigovresearch.com/huggingface-transformers-glossary-flashcards>`_
```
Comment (sgugger) [inline docs/source/glossary.rst:7]: While I love the idea of having flashcards for our glossary, I think this should be a bit further inside this page, after the definitions of the general terms.
Also, could you make it clearer it's not done by Hugging Face and take full credit for it? Something like "Darigov Research has worked on flashcards for ..."?

**Review by sgugger — COMMENTED** [review]
Thanks for the fixes and adding those flashcards! I just have a few comments on the wording.

**Comment by darigovresearch** [comment]
Thanks very much for the feedback, the relevant updates have been made as requested in a1295b7!

**Comment by darigovresearch** [comment]
@sgugger Is there anything else that is required for this pull request to be merged?

**Comment by darigovresearch** [comment]
@sgugger is there anything else that needs to be done on our side for this pull request to be merged?

**Comment by sgugger** [comment]
Hi @darigovresearch, sorry I missed your previous ping. We have just added a [community page](https://huggingface.co/transformers/master/community.html) in the documentation and we would actually prefer to put the links to your flashcards there if that's okay.

Sorry again for the delay!

**Comment by darigovresearch** [comment]
Hi @sgugger, no worries the updates have been made to the community.md file, the glossary.rst file still contains the corrections but has removed reference to the flashcards.

Is there anything else you need for this to be merged?

**Comment by sgugger** [comment]
Mope, that's perfect! Thanks a lot for your patience.

**Comment by darigovresearch** [comment]
@sgugger no worries & thanks for merging it!

When checking the page it appears that the rendering works in the .md file but not the final page - https://huggingface.co/transformers/master/community.html

Not sure what it could be, any thoughts?

Potentially add an extra blank line after the heading?

image

**Comment by sgugger** [comment]
Yes I just tested locally and it was the new line missing. I added it in [this commit](https://github.com/huggingface/transformers/commit/7acfa95afb8194f8f9c1f4d2c6028224dbed35a2) (directly on master).

**Comment by darigovresearch** [comment]
Great, thanks for the heads up and for the help!

**Pull request merged**